source/speech/sayAll.py

```diff
@@ -93,7 +93,9 @@ def isRunning(self):
 		"""
 		return bool(self._getActiveSayAll())
 
-	def readObjects(self, obj: "NVDAObjects.NVDAObject"):
+	def readObjects(self, obj: "NVDAObjects.NVDAObject", startedFromScript: bool = False):
```

Comment:
Same here

Please fix up the types and add a docstring for this function

```suggestion
	def readObjects(self, obj: "NVDAObjects.NVDAObject", startedFromScript: bool | None = False):
```